src/ansys/fluent/core/launcher/server_info.py

```diff
@@ -68,4 +86,6 @@ def _get_server_info(
     if not port:
         raise PortNotProvided()
 
+    _check_ip_port(ip=ip, port=port)
```

Comment:
The above check can be rolled into the new function - it is anyway a pre-requisite of testing the connection.

Reply:
Done. Thank you.